Library/ViewModels/DiscoveryViewModelTests.swift

```diff
@@ -394,96 +394,96 @@ internal final class DiscoveryViewModelTests: TestCase {
     self.vm.inputs.pageTransition(completed: true)
 
     XCTAssertEqual(
-      ["CTA Clicked", "CTA Clicked"], self.dataLakeTrackingClient.events,
+      ["CTA Clicked"], self.dataLakeTrackingClient.events,
       "Swipe event tracked once the transition completes."
     )
     XCTAssertEqual(
-      ["CTA Clicked", "CTA Clicked"], self.segmentTrackingClient.events,
+      ["CTA Clicked"], self.segmentTrackingClient.events,
       "Swipe event tracked once the transition completes."
     )
 
     XCTAssertEqual(
-      ["popular", "popular"], self.dataLakeTrackingClient.properties(forKey: "context_type"),
+      ["popular"], self.dataLakeTrackingClient.properties(forKey: "context_type"),
       "Correct sort is tracked."
     )
     XCTAssertEqual(
-      ["popular", "popular"], self.segmentTrackingClient.properties(forKey: "context_type"),
+      ["popular"], self.segmentTrackingClient.properties(forKey: "context_type"),
```

Comment:
I think you will have to re-visit this when you re-factor but i suspect the behavior i master *should* be the same expectation since all we are doing is re-factoring existing event logic.

Reply:
So the problem here is that, `discover_sort` means something different from what's on master, the ticket says it means previous sorted.